docs/dev/contribute.rst

```diff
@@ -116,7 +114,7 @@ Push your branch to GitHub:
 
   .. code-block:: bash
 
-      git push origin your_new_branch_name 
+      git push (optional: --set-upstream) origin your_new_branch_name 
```

Comment:
usually `[...]` is used to indicate optional arguments to a command, e.g. `[--set-upstream]`.

Reply:
thank you!!